models/Hand.java

```diff
@@ -22,10 +31,9 @@ public void claimTile(Tile claimedTile) {
     }
 
     // Discards a tile from Hand and returns it to the caller
-    public Tile discard(Tile discardedTile) {
+    public Tile discard() {
```

Comment:
This needs to accept a Tile named discardedTile and return the same tile (This will not have complete functionality yet but will allow the code to compile)

Reply:
Done.